README.md

```diff
@@ -75,8 +75,9 @@ async function test() {
   // => eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJhIjoxLCJiIjoyLCJjIjozLCJpYXQiOjE1Nzk1MjEyMTJ9.mIcxteEVjbh2MnKQ3EQlojZojGSyA_guqRBYHQURcfnCSSBTT2OShF8lo9_ogjAv-5oECgmCur_cDWB7x3X53g
 }
 
-// Using password protected private key
+// Using password protected private key - in this case you MUST provide the algorithm as well
```

Comment:
This should be part of the docs, not just in a comment. Do we have it in the options docs already? If not, let's add it there too

Reply:
Yes, it's there already:

> If the key is a passphrase protected private key, the algorithm option must be provided and must be either a RS* or ES* encoded key and the key option must be an object with the following structure:
>```js
>{
>  key: '<YOUR_RSA_ENCRYPTED_PRIVATE_KEY>',
>  passphrase: '<PASSPHRASE_THAT_WAS_USED_TO_ENCRYPT_THE_PRIVATE_KEY>'
>}
>```

I just realised that in the example I didn't put it in, that's all.